CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/ConfigurationPane.java

```diff
@@ -16,6 +16,7 @@
 package au.gov.asd.tac.constellation.plugins.importexport;
 
 import au.gov.asd.tac.constellation.graph.Attribute;
+import static au.gov.asd.tac.constellation.graph.attribute.interaction.AttributeValueTranslator.LOGGER;
```

Comment:
Doesn't look like this or the other added imports are being used. Could they be removed?